website/docs/docs/build/python-models.md

```diff
@@ -221,6 +241,29 @@ def model(dbt, session):
 
 </File>
 
+<VersionBlock firstVersion="1.8">
+    
+
+#### Dynamic configurations
+
+In addition to the existing methods of configuring Python models, you also have dynamic access to configuration values set with `dbt.config()` within Python models using f-strings. This expands the possibilities for custom logic and configuration management.
+
+<File name='models/my_python_model.py'>
+
+```python
+def model(dbt, session):
+    dbt.config(materialized="table")
+    
+    # Dynamic configuration access within Python f-strings, which allows for real-time retrieval and use of configuration values.
+    # Example: Dynamically accessing a configuration value
```

Comment:
sugg: conciseness, it's repeating the comment before it (ie no real info that's new and of high value) 

```suggestion
```